src/styles/blocks/about-us.scss

```diff
@@ -0,0 +1,85 @@
+.about-us {
+  margin: 100px 20px;
+
+  scroll-behavior: smooth;
+
+  @include on-tablet {
+    margin: 140px 72px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  @include on-desktop {
+    margin: 138px 120px 140px;
+  }
+
+  &__top {
+    margin-bottom: 8px;
+
+    &-description {
+      margin: 0;
+      font-size: 12px;
+      font-weight: 700;
+      color: #d12d35;
+    }
+
+    &-title {
+      margin: 0;
+      font-size: 32px;
+      font-weight: 700;
+
+      @include on-tablet {
+        font-size: 48px;
+        width: 210px;
+      }
+    }
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-top: 24px;
+    width: 277px;
+
+    @include on-tablet {
+      margin: 0;
+    }
+
+    @include on-desktop {
+      display: block;
+    }
+
+    &-schedule {
+      display: flex;
+      flex-flow: row wrap;
+      gap: 16px;
+    }
+
+    &-title {
+      margin: 16px 0;
+      font-size: 24px;
+      font-weight: 700;
+      text-align: left;
+    }
+  }
+
+  &--closed {
+    margin-top: 16px;
+  }
+
+  &--time {
+    font-weight: 700;
+
+    & > p {
+      margin-bottom: 8px;
+    }
+  }
+
+  &--day {
+    & > p {
```

Comment:
Avoid using html tags as selectors, if needed add other classes. [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Don%27t%20use%20tag%20names%20for%20styling)
Also, avoid styling an element depending on its relative position to another [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Styling%20an%20Element%20Depending%20on%20its%20Context)